performance-timeline/idlharness.html

```diff
@@ -75,8 +76,11 @@
   idl_array = new IdlArray();
   idl_array.add_untested_idls(document.getElementById("untested_idl").textContent);
   idl_array.add_idls(document.getElementById("idl").textContent);
+  // create first mark
+  self.performance.mark("mark");
   idl_array.add_objects({
-    Performance: ["window.performance"]
+    Performance: ["window.performance"],
+    PerformanceMark: [self.performance.getEntriesByName("mark")[0]]
```

Comment:
Why are we testing "mark"? So we can instantiate a PerformanceEntry?

Reply:
yes indeed. I need to instanciate one PerformanceEntry to trigger the toJSON test. I figured mark was the minimal one I could pick.